test/parser/mixins.spec.js

```diff
@@ -40,6 +40,16 @@ describe('Parser', () => {
         expect(root.first.first.empty).to.eql(true);
         expect(root.first.first.nodes).to.be.an('undefined');
       });
+
+      it('mixin without body and without whitespace #2', () => {
+        const root = parse('.base {.mixin-name}');
+
+        expect(root.first.first.type).to.eql('rule');
+        expect(root.first.first.selector).to.eql('.mixin-name');
+        expect(root.first.params).to.be.an('undefined');
+        expect(root.first.first.empty).to.eql(true);
+        expect(root.first.first.nodes).to.be.an('undefined');
```

Comment:
sure! There are 2 other instances or this same usage in a separate test in this file so I'm changing all of these in a follow up commit.